### PR TITLE
Hide edit button if clientId isn't set

### PR DESCRIPTION
**Pull request by mitchmac**

### CODE REVIEW

**Comment by spbyrne** [comment]
Perfect!

File: pages/_app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const App = ({ Component, pageProps }) => {
   return (
     <>
       <TinaEditProvider
-        showEditButton={true}
+        showEditButton={!!process.env.NEXT_PUBLIC_TINA_CLIENT_ID} // Don't show the edit button if there is no clientId.
```
Comment (jamespohalloran) [inline pages/_app.js:16]: Do we give the user any hints in the starter that they will need to add a client_id? With @ashleyjanedoiron 's PR, things are a little more graceful when a `client_id` isn't set: https://github.com/tinacms/teams-serverless/pull/715

Comment (mitchmac) [reply pages/_app.js:16]: @jamespohalloran I think we should, just wondering about the best way... Programmatically it makes sense (though probably rare) to not have client_id set when building a production site from the file system, so the conditions to show the hint would probably need to depend on more than if the client_id exists or not

Comment (jamespohalloran) [reply pages/_app.js:16]: To me, the ideal place would be to show that on the edit page, which gives the user context client_id's are used for the editing pages, and not the production pages. But in that case, we'd still want to show the edit-button. 
I'm also not sure if that jives with the default tinacms behaviour where the auth popup opens right away

**Comment by mitchmac** [comment]
Not a fan of this approach right now because it makes editing less discoverable when using the LocalClient. Will revisit.

**Pull request closed without merging**